### PR TITLE
Update Norwegian translation

### PR DESCRIPTION
**Pull request by zenseii**

### CODE REVIEW

**Comment by oleg-derevenetz** [comment]
Hi @zenseii what is the current status of this PR? Is it ready?

**Comment by zenseii** [comment]
Hi @oleg-derevenetz ! It is ready for review so I will try to ask if someone who knows Norwegian can review it first. If I get no reply we can just move ahead with it.

File: files/lang/nb.po [hunk 1]
```diff
@@ -185,8 +185,8 @@ msgid ""
 "The Moat reduces by -%{count} the defense skill of any unit and slows to "
 "half movement rate."
 msgstr ""
-"Vollgraven minsker forsvarsevnen til enhver styrke med -%{count} og den "
-"halverer flytteavstanden."
+"Vollgraven minsker forsvarsevnen til enhver styrke med -%{count} og halverer "
+"skrittene dens."
```
Comment (zenseii) [inline files/lang/nb.po:189]:
```suggestion
"skrittene deres."
```

File: files/lang/nb.po [hunk 2]
```diff
@@ -776,36 +776,36 @@ msgid ""
...
 msgid ""
 "Your enemies are allied against you and start close by, so be ready to come "
 "out fighting. You will need to own all four castles in this small valley to "
 "win."
 msgstr ""
-"Dine fiender er allierte mot deg og de starter i nærheten, så du må være "
-"klar for å havne i kamp. Du må eie alle de fire slottene i denne lille dalen "
-"for å vinne."
+"Dine fiender er allierte mot deg, og de starter i nærheten av deg, så du må "
+"være klar for kamp når som helst. Du må ha kontrollen over alle de fire "
```
Comment (zenseii) [inline files/lang/nb.po:796]:
```suggestion
"være klar for kamp når som helst. Du må kontrollere alle de fire "
```

File: files/lang/nb.po [hunk 3]
```diff
@@ -949,7 +948,7 @@ msgid ""
 "Capture all enemy castles for victory."
 msgstr ""
 "Dine ordrer er å slå ut landsherrene som har sverget å tjene Roland. Alle de "
-"fiendtlige slottene er forent imot deg. Siden du starter uten en borg må du "
+"fiendtlige borgene er forent imot deg. Siden du starter uten en borg må du "
 "skyndte deg med å ta over en før slutten av uka. Overta alle fiendtlige "
```
Comment (zenseii) [inline files/lang/nb.po:952]:
```suggestion
"skynde deg med å ta over en før slutten av uka. Overta alle fiendtlige "
```

File: files/lang/nb.po [hunk 4]
```diff
@@ -1084,60 +1084,60 @@ msgid ""
...
 msgid ""
 "Find your wayward son Joseph who is rumored to be living in the desolate "
 "lands. Do it before the first day of the third month or it will be of no "
 "help to your family."
 msgstr ""
-"Finn din villfarne sønn, Joseph, som ryktes å bo på de øde øyene. Gjør det "
-"før den første dagen av den tredje måneden ellers vil det være av ingen "
+"Finn din villfarne sønn, Josef, som ryktes å bo på de øde øyene. Gjør det "
+"før den første dagen av den tredje måneden, ellers vil det være av ingen "
 "nytte til familien din."
```
Comment (zenseii) [inline files/lang/nb.po:1104]:
```suggestion
"før den første dagen av den tredje måneden, ellers vil alt være til ingen "
"nytte for familien din."
```

File: files/lang/nb.po [hunk 5]
```diff
@@ -1084,60 +1084,60 @@ msgid ""
...
 
 msgid ""
 "Rescue your crazy uncle Ivan. Find him before the first day of the fourth "
 "month or it will be no help to your kingdom."
 msgstr ""
 "Redd din gale onkel Ivan. Finn han før den første dagen av den fjerde "
-"måneden ellers vil det være av ingen nytte til kongedømmet ditt."
+"måneden, ellers vil det være av ingen nytte til kongedømmet ditt."
```
Comment (zenseii) [inline files/lang/nb.po:1111]:
```suggestion
"måneden, ellers vil ikke kongedømmet ditt ha noe å tjene på det."
```

File: files/lang/nb.po [hunk 6]
```diff
@@ -1084,60 +1084,60 @@ msgid ""
...
-"grense! Ta tilbake dine frafalne byer og rykk inn i kongeriket i jungelen. "
-"Etterlat ingen overlevende."
+"Utslett barbarene som angriper kongedømmet ditt sin sørlige grense! Ta "
+"tilbake dine tapte byer, og rykk inn i kongeriket som befinner seg i "
+"jungelen. Ikke etterlat deg noen overlevende."
 
 msgid "Retake the castle of Ivory Gates, which has fallen due to treachery."
 msgstr ""
-"Gjenerobre borgen kalt Elfenbensportene, som har falt på grunn av forræderi."
+"Gjenerobre borgen kalt Elfenbensportene, som grunnet forræderi har falt."
```
Comment (zenseii) [inline files/lang/nb.po:1124]:
```suggestion
"Gjenerobre borgen kalt Ivory Gates, som har falt grunnet forræderi ."
```

File: files/lang/nb.po [hunk 7]
```diff
@@ -1146,18 +1146,18 @@ msgid "Power's End"
...
 
 msgid ""
 "Your mission is to vanquish the warring mages in the magical Shrouded Isles. "
 "The completion of this task will give you a fighting chance against your "
 "rivals."
 msgstr ""
-"Ditt oppdrag er å utslette de stridende magikerne i de magiske Skyggelagte "
-"Øyene. Fullføringen av denne oppgaven vil gi deg en sjanse mot motstanderne "
+"Ditt oppdrag er å utslette de krigende magikerne på De magiske skyggelagte "
+"øyene. Fullføringen av denne oppgaven vil gi deg en sjanse mot motstanderne "
```
Comment (zenseii) [inline files/lang/nb.po:1160]:
```suggestion
"øyene. Hvis du klarer denne oppgaven, vil du ha sjanse mot motstanderne "
```

File: files/lang/nb.po [hunk 8]
```diff
@@ -1173,23 +1173,23 @@ msgid ""
...
+"vil motstanderne dine utvilsomt ha kommet til den før deg."
 
 msgid ""
 "You must take control of the castle of Magic, where the fount of wizardry "
 "lies. Do this and your victory will be supreme."
 msgstr ""
-"Du må ta kontrollen over Magislottet som er hvor trolldomskilden ligger. "
-"Gjør dette og seieren din vil bli den aller største."
+"Du må ta kontrollen over Magislottet som er stedet hvor trolldomskilden "
+"ligger. Gjør dette, og seieren vil bli din aller største noen sinne."
```
Comment (zenseii) [inline files/lang/nb.po:1186]:
```suggestion
"ligger. Gjør dette, og seieren vil bli den største du noen sinne har oppnådd."
```

File: files/lang/nb.po [hunk 9]
```diff
@@ -1748,7 +1755,7 @@ msgid "Cannot recruit - guest to guard automove error."
 msgstr "Kan ikke verves. Feil ved flytting av besøkende til forsvarende."
 
 msgid "Cannot recruit - you already have a Hero in this town."
-msgstr "Helten kan ikke verves fordi du har allerede én i denne byen."
+msgstr "Helten kan ikke verves fordi det befinner seg en i denne byen."
```
Comment (zenseii) [inline files/lang/nb.po:1758]:
```suggestion
msgstr "Det kan ikke verves en helt fordi det befinner seg en i byen."
```

File: files/lang/nb.po [hunk 10]
```diff
@@ -2327,9 +2335,9 @@ msgid ""
 "trainer of gladiators agrees to train you in a skill of your choice."
 msgstr ""
 "Du går inn i arenaen og finner deg selv ansikt til ansikt med en flokk "
-"fryktelige løver. Du tar lett hånd om dem, til jubel og applaus fra "
-"publikum. En aldrende gladiatormester blir imponert over dine ferdigheter og "
-"går med på å trene deg opp i en ferdighet etter ditt ønske."
+"fryktelige løver. Du tar lett hånd om dem, og publikum roper i jubel. En "
+"aldrende gladiatormester ble imponert over prestasjonen din og går med på å "
```
Comment (zenseii) [inline files/lang/nb.po:2339]:
```suggestion
"aldrende gladiatormester ble imponert av prestasjonen din og går med på å "
```

File: files/lang/nb.po [hunk 11]
```diff
@@ -2679,7 +2686,7 @@ msgid "(digging ok)"
 msgstr "(graving mulig)"
```
Comment (zenseii) [inline files/lang/nb.po:2686]:
```suggestion
msgstr "(Graving mulig)"
```

File: files/lang/nb.po [hunk 12]
```diff
@@ -2988,16 +2995,16 @@ msgid "Manual"
...
 msgid "Toggle foreground sounds level."
-msgstr "Juster volumet til lydeffekter."
+msgstr "Juster volumet til lydeffekter og videosnutter."
 
 msgid "Change the type of music."
 msgstr "Skift typen musikk."
 
 msgid "Change the speed at which your heroes move on the main screen."
-msgstr "Skift hastigheten heltene dine flytter seg på på brettet."
+msgstr "Skift hastigheten heltene dine flytter seg rundt på brettet."
```
Comment (zenseii) [inline files/lang/nb.po:3007]:
```suggestion
msgstr "Skift hastigheten heltene dine flytter seg på brettet."
```

File: files/lang/nb.po [hunk 13]
```diff
@@ -3846,9 +3853,9 @@ msgid ""
 "This window provides information on the status of your hero or kingdom, and "
 "shows the date. Left click here to cycle through these windows."
 msgstr ""
-"Dette vinduet viser informasjon om tilstanden til helten eller kongedømmet "
-"ditt, samt viser datoen. Venstre-klikk her for å rullere mellom disse "
-"vinduene."
+"Dette vinduet viser informasjon om tilstanden til helten og kongedømmet "
+"ditt, i tillegg til dagens dato. Venstre-klikk her for å bla gjennom de "
+"forskjellige informasjonene."
```
Comment (zenseii) [inline files/lang/nb.po:3858]:
```suggestion
"forskjellige informasjonsvinduene."
```

File: files/lang/nb.po [hunk 14]
```diff
@@ -4144,7 +4150,7 @@ msgid "two"
 msgstr "2"
 
 msgid "You are unable to recruit at this time, your ranks are full."
-msgstr "Du kan ikke verve noen akkurat nå fordi hæren din har ikke plass."
+msgstr "Du kan ikke verve noen fordi hæren din ikke har plass."
```
Comment (zenseii) [inline files/lang/nb.po:4153]:
```suggestion
msgstr "Du kan ikke verve noen fordi hæren din har ikke plass."
```

File: files/lang/nb.po [hunk 15]
```diff
@@ -4241,35 +4247,35 @@ msgid "Treasure"
...
+msgstr "Du leter gjennom de fillete plaggene, men finner ingenting."
 
 msgid ""
 "You come across an old wagon left by a trader who didn't quite make it to "
 "safe terrain."
 msgstr ""
-"Du kommer over en gammel kjerre etterlatt av en byttehandler som ikke "
-"nøyaktig kom seg til trygg grunn."
+"Du kommer over en gammel varevogn etterlatt av en handelsmann, som etter alt "
+"å bedømme ikke hadde hellet med seg."
```
Comment (zenseii) [inline files/lang/nb.po:4260]:
```suggestion
"å bedømme ikke hadde hellet på sin side."
```

File: files/lang/nb.po [hunk 16]
```diff
@@ -4463,7 +4470,7 @@ msgstr ""
 "Flere slurker fra brønnen på én og samme dag vil ikke kunne hjelpe deg."
 
 msgid "A drink from the well has restored your spell points to maximum."
-msgstr "En slurk fra brønnen har fylt opp manaen din helt."
+msgstr "Du tar deg en slurk fra brønnen, og manaen din blir helt fylt opp."
```
Comment (zenseii) [inline files/lang/nb.po:4473]:
```suggestion
msgstr "Du tar deg en slurk fra brønnen, og manaen din blir fullstendig fylt opp."
```

File: files/lang/nb.po [hunk 17]
```diff
@@ -4473,21 +4480,21 @@ msgstr ""
 "allerede alt vi har å lære bort.\""
 
 msgid "The soldiers living in the fort teach you a few new defensive tricks."
-msgstr "Soldatene som bor i fortet lærer deg et par nye forsvarsgrep."
+msgstr "Soldatene som bor i fortet lærer deg et par nye forsvarstaktikker."
```
Comment (zenseii) [inline files/lang/nb.po:4483]:
```suggestion
msgstr "Soldatene som bor i fortet lærer deg et par nye forsvarstriks"
```

File: files/lang/nb.po [hunk 18]
```diff
@@ -4473,21 +4480,21 @@ msgstr ""
...
 
 msgid ""
 "You've come upon a mercenary camp practicing their tactics. \"You're too "
 "advanced for us,\" the mercenary captain says. \"We can teach nothing more.\""
 msgstr ""
-"Du kommer over en leiesoldatleir og de trener til strid. \"Du ligger for "
-"langt foran oss,\" sier kapteinen av leiesoldatene. \"Vi kan ikke lære bort "
-"noe mer.\""
+"Du kommer over en leiesoldatleir hvor det drives kamptrening. \"Du har et "
+"alt for høyt nivå for oss,\" sier leiesoldatenes kaptein. \"Det er ingenting "
```
Comment (zenseii) [inline files/lang/nb.po:4490]:
```suggestion
"mye høyere nivå enn oss,\" sier leiesoldatenes kaptein. \"Det er ingenting "
```

File: files/lang/nb.po [hunk 19]
```diff
@@ -4573,15 +4582,15 @@ msgstr ""
...
 "rett kurs."
 
 msgid ""
 "Your men spot a navigational buoy, confirming that you are on course and "
 "increasing their morale."
 msgstr ""
-"Mannskapet ditt får øye på et flytende sjømerke som bekrefter at du er på "
-"rett kurs og det styrker kampånden deres."
+"Mannskapet ditt får øye på et flytende sjømerke. Det bekrefter at dere er på "
+"rett kurs, og dette styrker kampånden deres."
```
Comment (zenseii) [inline files/lang/nb.po:4593]:
```suggestion
"rett kurs, og dette styrker kampånden til hele mannskapet."
```

File: files/lang/nb.po [hunk 20]
```diff
@@ -4711,20 +4720,23 @@ msgid ""
...
 "Gjennom en klarning får du øye på en eldgammel gjenstand. Dessverre er den "
-"bevoktet av 1 %{monster} som er rett ved. Ønsker du å kjempe mot 1 "
-"%{monster} for gjenstanden?"
+"bevoktet av 1 %{monster} like ved. Ønsker du å kjempe mot 1 %{monster} for "
+"gjenstanden?"
 
 msgid "Victorious, you take your prize, the %{art}."
-msgstr "Seirende tar du premien din, som er gjenstanden %{art}."
+msgstr ""
+"Triumferende griper du premien din, gjenstanden:\n"
```
Comment (zenseii) [inline files/lang/nb.po:4738]:
```suggestion
"Triumferende griper du premien din, som er gjenstanden:\n"
```

File: files/lang/nb.po [hunk 21]
```diff
@@ -5161,9 +5175,10 @@ msgid ""
 "you maps of the sea he made in his younger days for 1,000 gold. Do you wish "
 "to buy the maps?"
 msgstr ""
-"Kaptein som bor på denne godt vedlikeholdte fiskebrygga tilbyr deg et kart "
-"over hele havet som han lagde i sine yngre dager i bytte mot 1000 "
-"gullstykker. Ønsker du å kjøpe kartet?"
+"Kaptein som bor på denne godt vedlikeholdte fiskebrygga tilbyr deg et "
+"fullstendig sjøkart, som han lagde i sine yngre dager, i bytte mot 1000 "
```
Comment (zenseii) [inline files/lang/nb.po:5179]:
```suggestion
"komplett sjøkart, som han lagde i sine yngre dager, i bytte mot 1000 "
```

File: files/lang/nb.po [hunk 22]
```diff
@@ -6199,17 +6215,17 @@ msgid "morale|Blood!"
 msgstr "Blod!"
 
 msgid "Bad morale may cause your armies to freeze in combat."
-msgstr "Dårlig kampånd kan få styrkene dine til å stivne til i kamp."
+msgstr "Negativ kampånd kan få styrkene dine til å stivne til i kamp."
```
Comment (zenseii) [inline files/lang/nb.po:6218]:
```suggestion
msgstr "Negativ kampånd kan få styrkene dine til å stoppe opp i kamp."
```

File: files/lang/nb.po [hunk 23]
```diff
@@ -6199,17 +6215,17 @@ msgid "morale|Blood!"
...
 
 msgid ""
 "Neutral morale means your armies will never be blessed with extra attacks or "
 "freeze in combat."
 msgstr ""
-"Vanlig kampånd betyr at styrkene dine aldri verken vil bli velsignet med en "
+"Nøytral kampånd betyr at styrkene dine aldri verken vil bli velsignet med en "
 "ekstra tur, eller stivne til i kamp."
```
Comment (zenseii) [inline files/lang/nb.po:6225]:
```suggestion
"ekstra tur, eller stoppe opp i kamp."
```

File: files/lang/nb.po [hunk 24]
```diff
@@ -7512,8 +7528,8 @@ msgid ""
 "During a sudden storm, a bolt of lightning strikes a tree, splitting it. "
 "Inside the tree you find a mysterious mace."
 msgstr ""
-"Under en plutselig storm, slår et lyn ned i et tre og deler det i to. På "
-"innsiden av treet finner du en mystisk klubbe."
+"Midt i en uforventet storm, slår et lyn ned i et tre som deles i to. På "
```
Comment (zenseii) [inline files/lang/nb.po:7531]:
```suggestion
"Midt i en uforventet storm, slår et lyn ned i et tre, og det deles tvers gjennom. På "
```

File: files/lang/nb.po [hunk 25]
```diff
@@ -7564,14 +7580,14 @@ msgstr ""
...
-msgstr "Skjuleskjoldet av beskyttelse"
+msgstr "Tyvenes skjold av beskyttelse"
 
 msgid ""
 "A stone statue of a warrior holds a silver shield. As you remove the shield, "
 "the statue crumbles into dust."
 msgstr ""
-"En steinstatue av en kriger bærer et sølvskjold. I det du fjerner skjoldet "
-"faller statuen sammen i grus."
+"En statue av en kriger i stein bærer et sølvskjold. Idet du fjerner skjoldet "
```
Comment (zenseii) [inline files/lang/nb.po:7589]:
```suggestion
"En krigers statue i stein bærer et sølvskjold. Idet du fjerner skjoldet "
```

File: files/lang/nb.po [hunk 26]
```diff
@@ -7618,7 +7634,7 @@ msgid ""
 "your awaiting hands."
 msgstr ""
 "Rett foran deg dukker ei svevende glasskiste opp med en skriftrull lagt på "
-"en kongelig rød pute. I det du berører den, åpner lokket seg og rullen flyr "
+"en kongelig rød pute. Idet du berører den, åpner lokket seg og rullen flyr "
 "magisk inn i dine hender."
```
Comment (zenseii) [inline files/lang/nb.po:7638]:
```suggestion
"en kongelig rød pute. Idet du berører den, åpner lokket seg og rullen svever "
"magisk inn i utålmodige dine hender."
```

File: files/lang/nb.po [hunk 27]
```diff
@@ -7667,7 +7683,7 @@ msgid ""
 "ferociously, vanishing into thin air. Remembering the old leprechaun saying "
 "'Finders Keepers', you grab the sack and leave."
 msgstr ""
-"En liten regnbuegnom danser lykkelig rundt en magisk sekk. I det han ser deg "
+"En liten regnbuegnom danser lykkelig rundt en magisk sekk. Idet han ser deg "
 "komme stopper han i steget. Den lille mannen hyler og tramper med foten sin "
```
Comment (zenseii) [inline files/lang/nb.po:7687]:
```suggestion
"komme, stopper han midt i steget. Den lille mannen hyler og tramper med foten sin "
```

File: files/lang/nb.po [hunk 28]
```diff
@@ -7667,7 +7683,7 @@ msgid ""
...
 "febrilsk, før han forsvinner i løse luften. Du kommer på det gamle "
 "regnbuegnom ordtaket \"Finneren er vinneren\", og tar sekken og går din vei."
```
Comment (zenseii) [inline files/lang/nb.po:7689]:
```suggestion
"ordtaket til regnbuegnomene:  \"Finneren er vinneren\", og du tar sekken og går din vei."
```

File: files/lang/nb.po [hunk 29]
```diff
@@ -7706,10 +7722,10 @@ msgid ""
 "softest leather. Looking closely, you see fascinating ancient carvings "
 "engraved on the leather."
 msgstr ""
-"En forretningsnomade ber om beskyttelse fra en stamme med stridsvetter. Som "
-"takk for hjelpen din, gir han deg et par forseggjorte sko av det mykeste "
-"typen skinn. Ved nærmere blikk ser du forunderlige eldgamle inngraveringer "
-"på skinnet."
+"En forretningsnomade ber om beskyttelse fra en stamme med vetter. Som takk "
+"for hjelpen din, gir han deg et par forseggjorte sko av den mykeste typen "
+"skinn. Ved nærmere blikk ser du forunderlige eldgamle inngraveringer på "
```
Comment (zenseii) [inline files/lang/nb.po:7727]:
```suggestion
"skinn. Ved nærmere blikk oppdager du forunderlige eldgamle inngraveringer på "
```

File: files/lang/nb.po [hunk 30]
```diff
@@ -8487,7 +8503,7 @@ msgid ""
 msgstr ""
 "Du finner en detaljert beholder som rommer et gammelt pergament i "
 "kalveskinn. Runene på beholderen er veldig gamle, og håndverket som ligger "
-"bak den er fantastisk. I det du tar ut rullen føler du deg fylt med magisk "
+"bak den er fantastisk. Idet du tar ut rullen føler du deg fylt med magisk "
```
Comment (zenseii) [inline files/lang/nb.po:8506]:
```suggestion
"bak den er fabelaktig. Idet du tar ut rullen føler du deg fylt med magisk "
```

File: files/lang/nb.po [hunk 31]
```diff
@@ -8527,7 +8543,7 @@ msgid ""
 msgstr ""
 "Du kommer over et skilt. Det står: \"Her ligger kroppen til Anduran. Bøy deg "
 "og sverg troskap, og du vil bli belønnet.\" Du bestemmer deg for å gjøre det "
-"som står. I det du reiser deg føler du en kulde treffe huden din. Når du ser "
+"som står. Idet du reiser deg føler du en kulde treffe huden din. Når du ser "
```
Comment (zenseii) [inline files/lang/nb.po:8546]:
```suggestion
"som står. Idet du reiser deg, føler du en kulde treffe huden din. Når du ser "
```

File: files/lang/nb.po [hunk 32]
```diff
@@ -8663,15 +8679,15 @@ msgid ""
 msgstr ""
 "Du kommer over en slagmark hvor en ridderhelt har blitt dødelig såret av en "
 "gruppe zombier. Han ber deg om å ta hammeren hans og avslutte det han "
-"begynte. I det du plukker den opp begynner den å dure og så blir alt uklart. "
-"Zombiene ligger nå døde og det drypper blod fra hammeren. Du fester den til "
-"beltet ditt."
+"begynte. Idet du plukker den opp, begynner den å dure før ting blir tåkete "
```
Comment (zenseii) [inline files/lang/nb.po:8682]:
```suggestion
"begynte. Idet du plukker den opp, begynner den å dure før alt blir tåkete "
```

File: files/lang/nb.po [hunk 33]
```diff
@@ -8663,15 +8679,15 @@ msgid ""
...
+"for deg. Når du kommer til ligger zombiene døde, og det drypper blod fra "
```
Comment (zenseii) [inline files/lang/nb.po:8683]:
```suggestion
"for deg. Når du kommer til igjen, ligger zombiene døde, og det drypper blod fra "
```

File: files/lang/nb.po [hunk 34]
```diff
@@ -8680,8 +8696,8 @@ msgid ""
 "answers: \"You think this is funny? Fine. You can carry it. I much prefer "
 "flying anyway.\""
 msgstr ""
-"I det du kommer til toppen av en kolle kommer du over et latterlig syn. En "
-"fe prøver å bære et septer som er nesten like stort som den selv. Samtidig "
+"Idet du kommer til toppen av en kolle kommer du over et latterlig syn: en fe "
```
Comment (zenseii) [inline files/lang/nb.po:8699]:
```suggestion
"Idet du kommer til toppen av en kolle, kommer du over et latterlig syn: en fe "
```

File: files/lang/nb.po [hunk 35]
```diff
@@ -8680,8 +8696,8 @@ msgid ""
...
+"som prøver å bære et septer som er nesten like stort som seg selv. Samtidig "
 "som du prøver å holde deg fra å le spør du \"Trenger du hjelp?\" Feen "
```
Comment (zenseii) [inline files/lang/nb.po:8701]:
```suggestion
"som du prøver å holde deg fra å le, spør du \"Trenger du hjelp?\" Feen "
```

File: files/lang/nb.po [hunk 36]
```diff
@@ -8680,8 +8696,8 @@ msgid ""
...
 "stirrer bort på deg og svarer: \"Syns du dette er morsomt? Greit. Da kan du "
```
Comment (zenseii) [inline files/lang/nb.po:8702]:
```suggestion
"myser bort på deg og svarer: \"Syns du dette er morsomt? Greit. Da kan du "
```

File: files/lang/nb.po [hunk 37]
```diff
@@ -8933,7 +8948,7 @@ msgstr ""
 "forsinkelse"
 
 msgid "Slows all enemies to half movement rate."
-msgstr "Forsinker alle fiender ned til halvparten av flytteavstanden."
+msgstr "Forsinker alle fiender ned til halvparten av skrittene deres."
```
Comment (zenseii) [inline files/lang/nb.po:8951]:
```suggestion
msgstr "Reduserer alle fiender ned til halvparten av skrittene deres."
```

File: files/lang/nb.po [hunk 38]
```diff
@@ -9006,7 +9021,7 @@ msgstr "Antimagi"
 
 # en enkelt? flere?
 msgid "Prevents harmful magic against the selected creatures."
-msgstr "Forhindrer skadelig magi mot den utvalgte troppen."
+msgstr "Skjermer den utvalgte skapningen mot skadelig magi."
```
Comment (zenseii) [inline files/lang/nb.po:9024]:
```suggestion
msgstr "Skjermer den utvalgte skapningen fra skadelig magi."
```

**Comment by zenseii** [comment]
Hi @oleg-derevenetz ! When the checks are done, it would be great if you could have a look at whether this PR should be merged. Thank you!

**Comment by oleg-derevenetz** [comment]
Hi @zenseii Merged, thank you!

**Pull request merged**